libs/shared/src/types/notification.ts

```diff
@@ -153,22 +153,22 @@ export type NotifCategoryToNotifDataMapping = {
   [K in NotificationCategory]: K extends typeof NotificationCategories.NewComment
     ? INewCommentNotificationData
     : K extends typeof NotificationCategories.NewThread
-      ? INewThreadNotificationData
```

Comment:
I think your prettier version is outdated since these indents are recent additions.